cypress/integration/render-app.ts

```diff
@@ -0,0 +1,22 @@
+import { IFRAME_SELECTOR } from '../support/constants';
+
+describe('render app', () => {
```

Comment:
I think I would make this specific to a certain format. Like have:

- HTML Reader
  - Renders manager controls (this test)
  - Renders loading page while loading
  - Navigation (the tests in integration/navigations)
  - Display settings (for font family, size, etc)
- PDF Reader
  - ... same set of tests basically

Reply:
I've added a folder for the HTML reader tests, and also created a ticket to cover the rest of the tests.